SparseGrids/tasgridExternalTests.cpp

```diff
@@ -34,6 +34,7 @@
 
 #include "tasgridExternalTests.hpp"
 #include "tasgridTestHelpers.hpp"
+#include "tsgCoreOneDimensional.hpp"
```

Comment:
This file should already be visible here. Generally, I want to avoid redundant inclusions which helps keep things organized.

Reply:
Done. I removed this inclusion.